service/src/models/archaeologist.ts

```diff
@@ -30,10 +30,10 @@ export interface ArchaeologistInit {
   bootstrapList?: string[];
 }
 
-interface SarcophagusParameters {
+interface SarcophagusNegotiationParams {
   arweaveTxId: string;
   unencryptedShardDoubleHash: string;
-  maxRewrapInterval: number;
+  maxRewrapInterval: BigNumber;
```

Comment:
Is this guy meant to be a BigNumber? I'm thinking timestamp is safe to represent as a regular number